services/src/main/java/org/keycloak/broker/oidc/KeycloakOIDCIdentityProvider.java

```diff
@@ -94,7 +94,7 @@ public Response backchannelLogout(String input) {
             if (!validateAction(action)) return Response.status(400).build();
             if (action.getKeycloakSessionIds() != null) {
                 for (String sessionId : action.getKeycloakSessionIds()) {
-                    String brokerSessionId = provider.getConfig().getAlias() + "." + sessionId;
+                    String brokerSessionId = action.getIssuer() + "." + sessionId;
```

Comment:
When you change this, you'll need to migrate existing user sessions, at least the persistent user sessions.

Reply:
The problem is how to migrate this because we cannot assume the issuer is always available from the broker config.